File: library/src/main/java/net/robinx/queue/JobHandlerListener.java
package net.robinx.queue;

/**
 * Created by Robin on 2016/6/28 16:30.
 */
public interface JobHandlerListener<T extends Job<T>> {
    void onPrepare(T job);
    void onCancel(T job);
    void onFinish(T job);
}

File: library/src/main/java/com/github/jobqueue/JobHandlerListener.kt
package com.github.jobqueue

/**
 * Created by Robin on 2016/6/28 16:30.
 */
interface JobHandlerListener<T : Job<T>> {
    fun onPrepare(job: T)
    fun onCancel(job: T)
    fun onFinish(job: T)
}
